src/option.ts

```diff
@@ -27,5 +29,7 @@ export function parse(raw?: string): Options {
     }
     options[k] = value
   }
+  // @ts-ignore
+  global.no_namespace = options.no_namespace;
```

Comment:
Uh oh, this looks nasty.  is there any way we could avoid doing this? may be i could help if you can explain the reason why?

Reply:
This was just me fiddling/debugging and should not have made it into the commit.

That's why this value isn't read anywhere

Reply:
I'll remove it in the next round of changes

Reply:
Ah, all right. i was scared for a second there 🤪

Reply:
Resolving since this is removed in latest commit